workers/deployment_manager/GeneratedCode/GeneratedCode.csproj

```diff
@@ -0,0 +1,63 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <Platforms>x64</Platforms>
+    <Platform Condition="'$(Platform)' == ''">x64</Platform>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- Input properties for Improbable.CoreSdk.Tools -->  
+    <SchemaGeneratedOutputDir>$(SolutionDir)GeneratedCode\gen</SchemaGeneratedOutputDir>
+    <GeneratedDescriptorTarget>$(SolutionDir)tmp\schema.descriptor</GeneratedDescriptorTarget>
+    <GeneratedBundleTarget>$(SolutionDir)tmp\schema_bundle.json</GeneratedBundleTarget>
+
+    <GeneratedCodeDir>$(SolutionDir)GeneratedCode\sync</GeneratedCodeDir>
+  </PropertyGroup>
+
+  <!-- Input items for Improbable.CoreSdk.Tools -->
+  <ItemGroup>
+      <SchemaInputDir Include="$(SolutionDir)\..\..\schema;$(SolutionDir)DeploymentManager\schema" />
+  </ItemGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Improbable.CoreSdk" Version="13.5.0" />
```

Comment:
we can't unless you tie it to the project, which, as we discussed, we shouldn't do. There is also no reason to do that.
What we can do: have multiple releases for the deployment manager so that people can use the release with the correct version in it